providers/sparkpost/src/lib/sparkpost.provider.ts

```diff
@@ -62,6 +62,7 @@ export class SparkPostEmailProvider implements IEmailProvider {
 
     return {
       id: sent.results.id,
+      date: new Date().toISOString(),
```

Comment:
I see other mail providers returns current date, so I decided to do it here as well.

Reply:
@p-fernandez  SparkPost doesn't return timestamp